libs/govrn-contract-client/src/lib/govrn-contract-client.ts

```diff
@@ -78,4 +79,10 @@ export class GovrnContract {
   public async attestations(args: AttestationArgs) {
     return await this.govrn.attestations(args.tokenId, args.address);
   }
+
+  public async burnContribution(args: ContributionsArgs) {
+    return await this.govrn.burnContribution(args.tokenId, {
+      gasLimit: 2100000,
```

Comment:
@alexkeating is there any workaround for that?

Reply:
What happens if this is not specified?

Reply:
[`UNPREDICTABLE_GAS_LIMIT`](https://docs.ethers.io/v5/troubleshooting/errors/#help-UNPREDICTABLE_GAS_LIMIT) error.

Reply:
RInkeby